### PR TITLE
RavenDB-7239 Add multiple Urls tests

### PR DESCRIPTION
**Pull request by IdanHaim**

### CODE REVIEW

File: test/SlowTests/Bugs/ConnectionStringParsing.cs [hunk 1]
```diff
@@ -39,5 +39,26 @@ public void EnsureWellFormedConnectionStrings_Parsing_FailWithUnknownParameter()
             var dbParser = ConnectionStringParser<RavenConnectionStringOptions>.FromConnectionString("memory=true");
             Assert.Throws<ArgumentException>(() => dbParser.Parse());
         }
+
+        [Fact]
+        public void EnsureWellFormedConnectionStrings_ParsingMultiUrls_Successful()
+        {
+            var connectionStringParser = ConnectionStringParser<RavenConnectionStringOptions>.FromConnectionString("Urls=http://localhost:10301,http://localhost:10302,http://localhost:10303;");
```
Comment (ayende) [inline test/SlowTests/Bugs/ConnectionStringParsing.cs:46]: I thought the format was supposed to be:

`url=http://localhost:8080; url=http://localhost:8888`, etc, no?

Comment (ayende) [reply test/SlowTests/Bugs/ConnectionStringParsing.cs:46]: And database is required.

**Review by ayende — CHANGES_REQUESTED** [review]

File: test/SlowTests/Bugs/ConnectionStringParsing.cs (hunk 1 above)
Comment (IdanHaim) [reply test/SlowTests/Bugs/ConnectionStringParsing.cs:46]: The Format is going to change in this issue http://issues.hibernatingrhinos.com/issue/RavenDB-7286

Comment (ayende) [reply test/SlowTests/Bugs/ConnectionStringParsing.cs:46]: Please fix the format as part of this issue

**Pull request closed without merging**